docs/DeveloperGuide.md

```diff
@@ -148,6 +148,47 @@ The `Storage` component,
 
 Classes used by multiple components are in the `seedu.addressbook.commons` package.
 
+### Entity specific classes
+
+Yellowbooks has 3 primary entities: 'Person', 'Task', and 'Tag', these are represented by classes with the respective names. The 'Tag' class provides an association between a 'Person' and 'Task'.
+
+These are the specifications for the fields of 'Person' and 'Task'.
+
+#### Person
+
+Each field is implemented by a class with the same name. For example, the `Name` class represents the name of a person.
+
+Names of person(s) must be unique, names containing the same characters but in different cases are considered distinct. (e.g. Persons with name `John Doe` and `john doe` are considered different and hence allowed )
```

Comment:
Maybe it makes more sense for it to be case-insensitive?

Reply:
I'll add a id to person class